dart/shelf/pubspec.yaml

```
@@ -0,0 +1,8 @@
+name: shelf_app
+
+environment:
+  sdk: ^3.0.0
```

```suggestion
  sdk: ">=3.2.0 <3.3.0"
```
seems better to me, since we says that we run this code with dart3.2 (there will be PR for each update)